Cargo.toml

```
@@ -13,8 +13,8 @@ arc-swap = "1"
 compact_str = { version = "0.8", optional = true }
 crossbeam-queue = "0.3.11"
 dashmap = { version = "6", features = ["raw-api"] }
-hashlink = "0.9"
-hashbrown = "0.14.3"
+hashbrown = "0.15.2"
```

I think we only specified the patch version before because we used some `0.14.3` or newer features. It should now be okay to only specify major and minor.
```suggestion
hashbrown = "0.15"
```

---

Applied and edited the commit message.